website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -88,20 +111,191 @@ metrics: [String!]!
 dimension: String!
 ```
 
-### Metric value query parameters
+### Metric queries
+
+```graphql
+query(
+environmentId: BigInt!
+queryId: String!
+): QueryResult!
+```
+
 
-The mutation is `createQuery`. The parameters are as follows:
+**Create query**
 
 ```graphql
 createQuery(
 environmentId: Int!
 metrics: [String!]!
 dimensions: [String!] = null
 limit: Int = null
-startTime: String = null
-endTime: String = null
-where: String = null
+where: [String] = null
```

Comment:
@WilliamDee  is this correct now?

Reply:
Also, should we add Explain here?

Reply:
Yup, the where should be list now. still abit of discussion on the explain, we may plan on adding compileSql endpoint instead of explain.

Reply:
Ah cool! Should we create a task for this in linear? I did see the explain one but not the compile one.